tutorials/plot_receptive_field.py

```diff
@@ -0,0 +1,228 @@
+"""
+=====================================================================
+Spectro-temporal receptive field (STRF) estimation on continuous data
+=====================================================================
+
+This demonstrates how an encoding model can be fit with multiple continuous
+inputs. In this case, we simulate the model behind a spectro-temporal receptive
+field (or STRF). First, we create a linear filter that maps patterns in
+spectro-temporal space onto an output, representing neural activity. We fit
+a receptive field model that attempts to recover the original linear filter
+that was used to create this data.
+
+References
+----------
+Estimation of spectro-temporal and spatio-temporal receptive fields using
+modeling with continuous inputs is described in:
+
+.. [1] Theunissen, F. E. et al. Estimating spatio-temporal receptive
+       fields of auditory and visual neurons from their responses to
+       natural stimuli. Network 12, 289-316 (2001).
+
+.. [2] Willmore, B. & Smyth, D. Methods for first-order kernel
+       estimation: simple-cell receptive fields from responses to
+       natural scenes. Network 14, 553-77 (2003).
+
+.. [3] Crosse, M. J., Di Liberto, G. M., Bednar, A. & Lalor, E. C. (2016).
+       The Multivariate Temporal Response Function (mTRF) Toolbox:
+       A MATLAB Toolbox for Relating Neural Signals to Continuous Stimuli.
+       Frontiers in Human Neuroscience 10, 604.
+       doi:10.3389/fnhum.2016.00604
+"""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#
+# License: BSD (3-clause)
+
+import mne
+import numpy as np
+import matplotlib.pyplot as plt
+
+from scipy.stats import multivariate_normal
+from scipy.io import loadmat
+from mne.decoding import ReceptiveField, delay_time_series
+from sklearn.preprocessing import scale
+from sklearn.linear_model import Ridge
+from sklearn.metrics import r2_score
+rng = np.random.RandomState(1337)  # To make this example reproducible
```

Comment:
the only randomness comes from the noise added to the input data...if that's not enough reason to use `rng` then I'm happy to take it out and use `random.randn`